en/docs/references/ballerina-expression-syntax.md

```diff
@@ -492,3 +556,31 @@ To convert the `theRevenantJson` variable of type `json` to the record `Movie`,
 ```ballerina
 check theRevenantJson.cloneWithType(Movie);
 ```
+
+## XML
+
+XML is a textual format for representing a string or an XML object. The following topics describe these values.
+
+### String values in XML
+
+Ballerina allows you to cast a string value in XML via variables of type `xml`.
+
+###### Examples:
+
+To initialize a variable of type `xml`, you can enter an expression as follows:
+
+```ballerina
+    xml `"Dear customer,"`
+```
+
+### XML objects
+
+You can define an XML object to hold XML values.
```

Comment:
I think we have to use the proper XML terms here.

Reply:
Rewrote this section.